recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - toolchain
     - gcc          # [osx]
     - python
+    - setuptools
```

Comment:
Discovered this was using `setuptools` after re-rendering. See this [build]( https://circleci.com/gh/conda-forge/scipy-feedstock/78 ) as evidence. As a result, we have added it here and the need arguments above.